static/simple-portfolio-template/index.html

```diff
@@ -0,0 +1,344 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="favicon.svg" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;500;600;800&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./css/style.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>A Simple Portfolio Template</title>
+    <script type="module" defer src="./js/main.js"></script>
+  </head>
+
+  <body>
+    <section
+      class="bg-hero-bg relative bg-cover bg-fixed bg-center text-white"
+      id="hero-section"
+    >
+      <article class="navbar">
+        <div class="flex items-center justify-between">
+          <h3 class="pt-2 text-3xl font-semibold uppercase">Willsim</h3>
+
+          <div class="pt-2 lg:hidden">
+            <button>
+              <img
+                src="./assets/images/menu.png"
```

Comment:
For icons, we should use SVG format when possible. They are much smaller and don't loose quality at any size.